.gitignore

```diff
@@ -24,7 +24,10 @@ var/
 .vscode/settings.
 
 # Generated files
+csrc/flash_attn_rocm/src/*hip*
```

Comment:
beter to use *_hip*?

Reply:
makes sense

Reply:
Yes, file names such as ```hip_flash_attention.cpp``` are ignored too?